src/screens/ExchangeModal.tsx

```diff
@@ -348,6 +358,18 @@ export default function ExchangeModal({
     }
   }, [currentNetwork, prevTxNetwork]);
 
+  useEffect(() => {
+    const getNativeAsset = async () => {
+      if (!outputNetwork || !accountAddress) return;
+      const nativeAsset = await ethereumUtils.getNativeAssetForNetwork(
+        outputNetwork,
+        accountAddress
+      );
+      setOutputNetworkDetails(nativeAsset);
+    };
+    getNativeAsset();
```

Comment:
can we call this `getNativeOutputAsset` or something like that? did a double take

Reply:
+1